package.json

```diff
@@ -9,7 +9,10 @@
     "PEG.js"
   ],
   "homepage": "https://peggyjs.org/",
-  "repository": "peggyjs/peggy",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/peggyjs/peggy.git"
+  },
```

Comment:
This is a new preferred way to specify repositories? What the advantages over the previous format?

Reply:
Another great question.  If you do `npm pkg fix` it will switch your repository to this format.  This format is what is used by the provenance mechanism of `npm publish`.  If you use the simpler format, npm will make this change at publish time and give a warning telling you to run `npm pkg fix`.